README.md

```diff
@@ -141,9 +141,9 @@ A `RequirementType` of `All` changes the traversal. First, if there are no filte
 
 In the above example, `FeatureW` specifies a `RequirementType` of `All`, meaning all of it's filters must evaluate to true for the feature to be enabled. In this case, the feature will be enabled for 50% of users during the specified time window.
     
-### Service Registration
+## Service Registration
 
-Feature flags rely on .NET Core dependency injection. We can register the feature management services using standard conventions.
+Feature flags rely on .NET Core dependency injection. We can register the feature management services using standard conventions. By default, the feature manager and feature filters should be registered as singleton services by calling `AddFeatureManagement`.

```

Comment:
I'd leave out the newly added text to keep the simple case simple.

I'd suggest any new content or clarifications be a part of the new section(s).

Reply:
Sounds good.